src/index.html

```diff
@@ -7,6 +7,6 @@
     <title>Code wars!</title>
 </head>
 <body style="height: 100%">
-  <div id="app" style="height: 100%"></div>
+  <div id="app" style="height: 100%; background-color: #303133;"></div>
```

Comment:
Please add className to avoid some inline logic

Reply:
Will be fixed